ext/storage_sources/s3_store/test/test_aws_bucket_conn.cpp

```diff
@@ -14,16 +14,18 @@ namespace test_defaults
 int test_list_buckets(const Aws::S3Crt::ClientConfiguration &config);
 
 /* Wrapper for unit test functions. */
-#define TEST(func, config, expected)                   \
-    do {                                               \
-        int __ret;                                     \
-        if ((__ret = (func(config))) != expected)      \
-            return (__ret);                            \
+#define TEST(func, config, expected_output)                   \
+    do {                                                      \
+        int __ret;                                            \
+        if ((__ret = (func(config))) != expected_output)      \
+            return (__ret);                                   \
     } while (0)
 
 /*
  * test_list_buckets --
- *     Unit test for listing S3 buckets under the associated AWS account.
+ *      Example unit test for listing S3 buckets under the associated AWS account.
```

Comment:
I would rephrase it to:
`Example of a unit test to list S3 buckets under the associated AWS account.`